reana-ui/src/util.js

```diff
@@ -190,7 +197,15 @@ export function parseLogs(logs) {
   const parsedLogs = JSON.parse(logs);
 
   for (let job of Object.values(parsedLogs.job_logs)) {
-    job.duration = formatDuration(getDuration(job.started_at, job.finished_at));
+    if (job.status === "stopped") {
+      // Hide the duration of the job, because in the job object
+      // there is no info about when the job (or the workflow) was stopped.
+      job.duration = "";
```

Comment:
```suggestion
      job.duration = null;
```

What about `null` instead of `""`?  We already set the duration to null if it is not available or it is not valid:
https://github.com/reanahub/reana-ui/blob/78153048f3479c32d56b46ddb0343b1af9fa4b64/reana-ui/src/util.js#L118-L127